revenuecatui/src/iosMain/kotlin/com/revenuecat/purchases/kmp/ui/revenuecatui/UIKitInteropPropertiesKtx.kt

```diff
@@ -4,26 +4,38 @@ import androidx.compose.ui.ExperimentalComposeUiApi
 import androidx.compose.ui.viewinterop.UIKitInteropProperties
 
 /**
- * Uses NonCooperative UIKitInteropInteractionMode if available, for snappy scrolling.
+ * Set UIKitInteropProperties without having to opt-in to experimental APIs.
+ *
+ * @param nonCooperativeInteractionMode Uses NonCooperative UIKitInteropInteractionMode if
+ * available, for snappy scrolling.
+ * @param isNativeAccessibilityEnabled Enables the native accessibility features of the underlying
+ * UIKit view.
  */
 @OptIn(ExperimentalComposeUiApi::class)
-internal fun nonCooperativeUiKitInteropPropertiesNonExperimental(): UIKitInteropProperties =
-    try {
+internal fun uiKitInteropPropertiesNonExperimental(
+    nonCooperativeInteractionMode: Boolean,
+    isNativeAccessibilityEnabled: Boolean,
```

Comment:
Wondering, should we set default values of these to `true`? Since that's probably what we would like most of the time.

Reply:
Sounds good, thanks for explaining!